src/form/qust.rs

```diff
@@ -211,7 +211,7 @@ impl TESQuest {
                 for log in log_entries.iter() {
                     let stage = index.stage;
                     let log_string_offset = log.string_offset;
-                    if log_string_offset == 4294967295 {
+                    if log_string_offset == 4294967295 {//that's 2^32 => prevent overflow, probably
```

Comment:
It is because invalid data is specially marked as 0xffffffff

Reply:
that was more a mental note for myself when looking at the code looking for offsets that would need updating, I forgot about putting that comment in.